### ISSUE TITLE
Error: Only sources that implement IAsyncEnumerable can be used for Entity Framework asynchronous operations

### ISSUE THREAD

**Issue by holyrong** [abpframework/abp#4890]
abp vnext version **3.0.4**

I want to create a GET api to return a list，I've tried two ways to implement it，one is successful and another is failed。

1.the failed code like belows:

```
        protected override IQueryable<Group> CreateFilteredQuery(GroupRequestDto input)
        {
            IQueryable<Group> query = Repository.WithDetails(cs => cs.Subject);
            query = query.Where(cs => cs.Id == input.Id);

            return query;
        }
```

I've got the follow **error messages**:

> System.InvalidOperationException: The source IQueryable doesn't implement IAsyncEnumerable<Group>. Only sources that implement IAsyncEnumerable can be used for Entity Framework asynchronous operations.

2.the successful way:

```
        public async Task<IList<GroupReadDto>> GetGroups(GroupRequestDto input)
        {
            if (input == null || !input.Id.HasValue)
            {
                return await Task.FromResult(new List<GroupReadDto>());
            }

            IQueryable<Group> query;

            query = Repository.WithDetails(cs => cs.Subject);
            query = query.Where(cs => cs.Id == input.Id);

            var list = ObjectMapper.Map<IQueryable<Group>, IList<GroupReadDto>>(query);

            return await Task.FromResult(list);
        }
```

**Comment by liangshiw** [comment]
Solved  By https://github.com/abpframework/abp/issues/4806

**Comment by holyrong** [comment]
@liangshiw 
**do you mean it will be solved in version 3.1** ?

**Comment by maliming** [comment]
Yes, it's will available at v3.1.
